phpstan-baseline.neon

```diff
@@ -35,6 +35,11 @@ parameters:
 			count: 1
 			path: config/drupal-phpunit-bootstrap-file.php
 
+		-
+			message: "#^Else branch is unreachable because ternary operator condition is always true\\.$#"
+			count: 1
+			path: src/Rector/AbstractDrupalCoreRector.php
+
```

Comment:
If we add the following to `phpstan.neon` this won't be reported:

```
	dynamicConstantNames:
		- Drupal::VERSION
```

See https://github.com/mglaman/phpstan-drupal/blob/main/extension.neon#L14-L15

Reply:
Fixed